app/views/layouts/application.html.erb

```diff
@@ -12,52 +12,13 @@
 </head>
 
 <body>
+<%= render "layouts/header" %>
 
-  <div class="top-bar">
-    <span class="eff-branding">(logo) A Project of the Electronic Frontier Foundation</span>
-    <span class="search">
-      Search&nbsp;
-      <input type="text">
-    </span>
-  </div>
-  <header>
-    <span class="logo">Logo + Title</span>
-  </header>
-
-
-  <% if breadcrumbs.present? %>
-  <div id="breadcrumbs">
-    <%= render_breadcrumbs %>
-  </div>
-  <% end %>
-
-    <div id="main-menu">
-      <ul>
-        <li><%= link_to "Lessons", topics_path %></li>
-        <li><%= link_to "Blog", blog_path %></li>
-        <li><%= link_to "Training 101", articles_path %></li>
-        <li><%= link_to "Materials", materials_path %></li>
-      </ul>
-    </div>
     <div id="main-content">
       <%= yield %>
     </div>
 
-  <footer>
-    <div class="row">
-      Footer
-    </div>
-  </footer>
-  <%= javascript_include_tag "application" %>
-
-  <div id="anon-stats">
```

Comment:
Can you restore this div? As well as the `#anon-stats` rule from layout.scss.

Reply:
Sorry about that! I got delete happy when removing the old footer div/styles.